.github/workflows/nightfalldlp.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: nightfallDLP action step
-        uses: nightfallai/nightfall_dlp_action@v0.0.2
+        uses: nightfallai/nightfall_dlp_action@FixInitialCommitDiff
```

Comment:
do you think there's a way we can have this always point to the current branch (so it will always run and we don't have to worry about constantly updating it)? Wondering if there's an easy env var that Github gives us we coulde point to: https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
`$GITHUB_HEAD_REF` would be the value to use, but it looks like using env vars is not supported in the `uses:` clause https://github.community/t/env-variables-in-uses/17466

Reply:
It looks like you can use `v1` this would cover all releases in `v1.x.x`

Reply:
Would it use the most recent tag then?

Reply:
ahh, I believe people use a general tag like `v1` and they move the tag as they want to release newer releases

Reply:
Yea I think josh was wondering if there was a simpler way to point the action version to the current branch so we wouldn't have to update it to like `FixInitialCommitDiff`/ PR branch name when we're testing/developing, but I don't think it's possible since we can't use env vars in the `uses` section.